test/JacobianTest.jl

```diff
@@ -279,4 +279,18 @@ end
     end
 end
 
+@testset "BigFloat" begin
```

Comment:
Does this test cover all the new branches? It seems CodeCov is sleeping at the wheel.
I think we may also want to add cases where only some of the values are uninitialized?

Reply:
Codecov confirms that all modified lines are covered by the tests: https://app.codecov.io/github/JuliaDiff/ForwardDiff.jl/commit/ec3e483747159b46e0db5328c60c3543c13478a6/indirect-changes